### ISSUE TITLE
Possible out

### ISSUE THREAD

**Issue by thynson** [tim-smart/multipasta#22]
https://github.com/tim-smart/multipasta/blame/451de6a06ee4b1d05b5b6903e84ce97b0a277a19/src/internal/headers.ts#L94

**Comment by tim-smart** [comment]
It's fine in this case as out of bound reads return undefined, and the start < end check will return false in the next phase of parsing.

**Comment by thynson** [comment]
> It's fine in this case as out of bound reads return undefined, and the start < end check will return false in the next phase of parsing.

You're right. It's fine in Javascript, I must have been suffered too much with recent experience of C and Rust 😆 .
